cmd/manager/aggregator.go

```diff
@@ -373,8 +378,8 @@ func createResults(crClient *complianceCrClient, scan *compv1alpha1.ComplianceSc
 		if canCreate, why := canCreateRemediation(scan, remTargetObj); !canCreate {
 			// Only issue event once.
 			if !remediationNotPossibleEventIssued {
-				log.Info(why)
-				crClient.recorder.Event(scan, v1.EventTypeWarning, "CannotRemediate", why)
+				log.Info(why + " Remediation:" + crkey.Name)
```

Comment:
Let's use the `"key": "value"` format to log this, this way it'll show up as JSON in the logs:

It would look like this:

```
log.Info(why. "Remediation", crkey.Name)
```